tools/run_tests/python_utils/dockerjob.py

```diff
@@ -21,8 +21,11 @@
 import tempfile
 import time
 import uuid
+import sys
 
-from . import jobset
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+
```

Comment:
Extra space? What would YAPF recommend?

Reply:
Looks like yapf just preserves whatever spacing you have in this case. Removed.